src/Teams/Services/IManageTeamsService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace Teams.Services
+{
+    public interface IManageTeamsService
+    {
+        Task<bool> Remove(int team_id);
```

Comment:
RemoveAsync please, we should stick to naming convention. If the method returns Task then the method name should end with Async